docs/content/modeling/roles-and-permissions.mdx

```diff
@@ -21,16 +21,16 @@ import {
 
 <DocumentationNotice />
 
-In this guide you will learn how to model roles and permissions model within <ProductName format={ProductNameFormat.ProductLink}/> using the _<ProductConcept section="what-is-an-authorization-model" linkName="authorization model" />_ and _<ProductConcept section="what-is-a-relationship-tuple" linkName="relationship tuple" />_.
+This guide explains how to model roles and permissions in <ProductName format={ProductNameFormat.ProductLink}/> using <ProductConcept section="what-is-an-authorization-model" linkName="authorization models" /> and <ProductConcept section="what-is-a-relationship-tuple" linkName="relationship tuples" />. For the purpose of this guide:
```

Comment:
I don't love the idea of starting a how-to guide with "This guide explains how to...". You can just strike this and start with, "To model roles and permissions..."